pom.xml

```diff
@@ -303,7 +303,7 @@
                         </sources>
                     </configuration>
                 </execution>
-          </executions>          
+          </executions>
```

Comment:
spurious, but also unnecessary trailing. Can separate this if deemed useful, just thought there weren't too much of these so left them.